discovery_engine/test/ffi/ffi_test.dart

```diff
@@ -13,15 +13,13 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import 'package:test/test.dart';
-import 'package:xayn_discovery_engine/discovery_engine.dart' show asyncCore;
 
 import '../logging.dart' show setupLogging;
 
 void main() {
   setupLogging();
 
-  test('calling async ffi functions works', () async {
-    final x = await asyncCore.add(10, 22);
-    expect(x, equals(32));
+  test('calling async ffi functions works', () {
+    // TODO
```

Comment:
What forbid us to start writing some test?

Reply:
we could indeed take the raw ffi from the genesis extension file for this test. but since we don't use primitives to call the ffi anymore, there is more setup required, which is handled in the dart wrapper of the follow up pr anyways.